firmware/lib/screenManager/screenManager.cpp

```diff
@@ -14,6 +14,12 @@ ScreenManager::ScreenManager(TFT_eSPI &tft) : m_tft(tft) {
   m_tft.setTextDatum(MC_DATUM);
   reset();
 
+  // I'm not sure which cache size is actually good.
+  // Needs testing.
+  m_render.setCacheSize(128, 128, 8192);
+  setFont(DEFAULT_FONT);
+  m_render.setDrawer(m_tft);
```

Comment:
OpenFontRenderer probably also has implementations for other `Drawers` or whatever they call it,
would it make sense to strip those files from this PR?

Reply:
I don't think there are specific files for the other drawers (it's more like an interface definition calling the same functions like .drawPixel() ).